client/routes/Routes.jsx

```diff
@@ -60,31 +60,81 @@ const Routes = () => {
             <InterceptAnonymizableRoute path="/run/:scenarioId">
                 <Route component={Run} />
             </InterceptAnonymizableRoute>
+
             <ConfirmAuth
-                path="/editor/new"
-                requiredPermission="create_scenario"
+                exact
+                path="/account-administration"
+                requiredPermission="edit_permissions"
             >
-                <Route component={NewScenario} />
+                <Route component={AccountAdmin} />
             </ConfirmAuth>
             <ConfirmAuth
-                path="/editor/:id"
-                requiredPermission="create_scenario"
+                exact
+                path="/cohorts"
+                requiredPermission="view_run_data"
             >
-                <Route component={Editor} />
+                <Route
+                    component={props => (
+                        <Facilitator {...props} activeTab="cohorts" />
+                    )}
+                />
             </ConfirmAuth>
             <ConfirmAuth
                 exact
-                path="/cohorts"
+                path="/facilitator/cohorts"
```

Comment:
is this and the `/facilitator/search` path used anywhere else?

Reply:
I'm not quite sure yet. Since the meeting yesterday, I have to rethink the way I'm organizing the URLs, so this is likely to completely change